### PR TITLE
Add install target

### PR DESCRIPTION
**Pull request by Alexander-Wilms**

### CODE REVIEW

**Comment by TingPing** [comment]
This should be done in scons.

**Comment by guillaumechereau** [comment]
I don't mind doing this in the Makefile, but if there is a way to limit the lines size to 80 chars?
If not I rather put this in scons directly.

File: Makefile [hunk 1]
```diff
@@ -30,3 +29,23 @@ doc:
...
 	naturaldocs -i /tmp/goxel_src -o html doc -p ndconfig
+
+PREFIX ?= /usr/local
+
+.PHONY: install
+install:
+	install -Dm744 goxel $(DESTDIR)$(PREFIX)/bin/goxel
+	for size in 16 24 32 48 64 128 256; do \
+		install -Dm644 data/icons/icon$$size.png $(DESTDIR)$(PREFIX)/share/icons/hicolor/$${size}x$$size/apps/goxel.png; \
+	done
```
Comment (TingPing) [inline Makefile]: Run `gtk-update-icon-cache -ftq $(DESTDIR)$(PREFIX)/share/icons/hicolor`

Comment (Alexander-Wilms) [reply Makefile]: Is this also required for KDE?

Comment (TingPing) [reply Makefile]: No, KDE has its own caching mechanism.

**Comment by guillaumechereau** [comment]
I merged the commit manually.

**Comment by Alexander-Wilms** [comment]
Thanks

**Pull request closed without merging**